package.json

```diff
@@ -37,6 +37,7 @@
         "ember-cli-htmlbars": "^6.0.1",
         "ember-cli-sass": "^10.0.1",
         "ember-cli-typescript": "^5.1.0",
+        "ember-on-resize-modifier": "^1.0.0",
```

Comment:
this should be fine, but another option in cases like this is to put it in `peerDependencies`, especially if it's a package that might get used directly in our consuming apps. As that allows more flexibility, and allows the parent app to determine what version to use if it is using it too and if there is a conflict.

Reply:
Never heard of `peerDependencies` before but after a quick read, that makes the most sense. Was just going off of the docs here: https://cli.emberjs.com/release/writing-addons/addon-dependencies/ to ensure that any deps required by the addon are in `dependencies`.

Reply:
I could think of a couple projects that used it, thats the only reason i mentioned it. But yeah, for the most part I think having it in main `dependencies` is the right call. Just wanted to note that more for future case, if we have something that is used all over the place. 

I *think* w/`peerDependencies` yarn/npm will automatically install it too, if you dont explicitly declare it in the parent app's package.json, and just throws a warning.